apps/web/src/components/PoolCandidatesTable/PoolCandidatesTable.tsx

```diff
@@ -426,7 +426,7 @@ const PoolCandidatesTable = ({
       {
         id: "status",
         header: intl.formatMessage(tableMessages.status),
-        enableHiding: false,
+        enableHiding: !recordOfDecision, // After record of decision is turned on, we can remove this property entirely (it defaults to true)
```

Comment:
Should it not just be `enableHiding: recordOfDecision` ?
Now RoD as false resolves to true making it hide-able 

Does not meet goal
"Status column should not be hideable if RoD is false"